src/ticket.c

```diff
@@ -955,6 +987,12 @@ static void next_action(struct ticket_config *tk)
 				ticket_activate_timeout(tk);
 			}
 		}
+	} else {
+		/* for manual tickets, just send ACKs if needed */
+		if (tk->acks_expected) {
+			handle_resends(tk);
```

Comment:
Resends are handled, IIRC, only by the side which initiates the operation. So, I wonder if this is necessary for manual tickets.